### DISCUSSION TITLE
Docs suggestion: guide to where things go in the `.yml` file

### DISCUSSION THREAD

**Discussion by jtr13** [quarto-dev/quarto-cli#5803] [Feature Requests]
### Description

I've been struggling a big figuring out where options go in the `.yml` -- for example on this page https://quarto.org/docs/reference/formats/html.html there is a complete list of options but it's not always clear which key the options should go under. I've been mainly looking at examples to figure it out.

**Comment by jtr13** [comment]
Also discussed in the following issues:

https://github.com/quarto-dev/quarto-cli/issues/5186 (`book:` vs `format:` options)

https://github.com/quarto-dev/quarto-cli/issues/4315 (`fig-align` doesn't work in `html: format:`)

https://github.com/quarto-dev/quarto-cli/issues/3371 (`knitr: opts_chunk:` takes `.` format)

**Comment by mcanouil** [comment]
(Changing this to Feature Requests)

**Comment by cwickham** [comment]
Linking to Reference Improvement issue:  #5187

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/5187
